frontend/test/metabase/scenarios/collections/permissions.cy.spec.js

```diff
@@ -121,6 +121,30 @@ describe("collection permissions", () => {
               });
             });
           });
+
+          onlyOn(permission === "view", () => {
+            it.skip("should not be offered to save dashboard in collections they have `read` access to (metabase#15281)", () => {
+              cy.signIn(user);
+              cy.visit("/");
```

Comment:
If you go to `/collection/root` or any collection that you don't have curate permissions for, and then click the "New dashboard", then it will by default pre-select the collection you're in - without the need to change anything.

Reply:
Yeah, the problem with it is that the collection I'm in is the root collection where I also don't have curate access in. I'll add that case as well.

Reply:
@flamber updated the test and also description/screenshots